Polyhedron/demo/Polyhedron/Polyhedron_demo_detect_sharp_corners.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
```

Comment:
these functions should be moved to the `Detect_sharp_edges_plugin` files too, in the `Plugins/PMP` folder